src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,58 @@
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 706px;
+  padding: 10px;
+  gap: $gap-size;
+  margin: 0 auto;
```

Comment:
since you center it properly with the help of body, you don't need margin and padding